src/components/orders/OrderNotFound/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { useParams } from 'react-router'
 import styled from 'styled-components'
-import { Search } from 'components/orders/Search'
+import { Search } from 'apps/explorer/components/common/Search'
```

Comment:
With this change, we might be missing some CSS. 
![image](https://user-images.githubusercontent.com/622217/137188003-41a29ec0-706a-458c-968d-1e03022f1866.png)
The search box should be aligned to the left. 

![image](https://user-images.githubusercontent.com/622217/137188258-b2e802c5-5e6c-444a-afa9-ad3985892881.png)